{{ cookiecutter.project_name }}/webpack_dev_server.sh

```diff
@@ -1,36 +1,21 @@
 #!/bin/bash
 set -ef -o pipefail
 
-if [[ $(uname -s) == "Darwin" ]] ; then
-  IS_OSX_HOST_MACHINE="true"
-else
-  IS_OSX_HOST_MACHINE="false"
-fi
-
-if [[ "$IS_OSX_HOST_MACHINE" == 'true' ]] ; then
-  WEBPACK_HOST='127.0.0.1'
-elif [ -z "$WEBPACK_DEV_SERVER_HOST" ] ; then
-  WEBPACK_HOST='0.0.0.0'
-else
-  WEBPACK_HOST="$WEBPACK_DEV_SERVER_HOST"
-fi
+# Define some environment variables to figure out where we are running
+source ./scripts/envs.sh
 
-if [ -z "$WEBPACK_DEV_SERVER_PORT" ] ; then
-  WEBPACK_PORT='{{ cookiecutter.webpack_dev_port }}'
-else
-  WEBPACK_PORT="$WEBPACK_DEV_SERVER_PORT"
-fi
+WEBPACK_HOST='0.0.0.0'
+WEBPACK_PORT='8078'
```

Comment:
In cookiecutter.json, webpack_dev_port is set to 8072 and when I load the index page, 'style.js' and 'root.js' javascript files are successfully loaded from http://0.0.0.0:8072 but '__webpack_hmr' tries to load from http://0.0.0.0:8078 and fails (refused to connect).  If I change the above to 8072 then it loads successfully.